content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -125,24 +99,24 @@ This creates a Prisma schema in `prisma/schema.prisma`.
 
 **Note:**<br /><br />
 
-This process also creates an `.env` file, but this file has no effect when you use Cloudflare Workers.
+`prisma init` also creates an `.env` file, but this file has no effect when you use Cloudflare Workers.
```

Comment:
This is semi misleading as the tutorial uses `prisma migrate` below, which will read the file to be able to figure out what database to talk to. This will currently fail.

Reply:
I've updated the note here.